addOns/graphql/src/main/java/org/zaproxy/addon/graphql/ExtensionGraphQl.java

```diff
@@ -106,37 +111,79 @@ public String getDescription() {
                         1,
                         null,
                         "",
-                        "-graphqlfile <path>      "
+                        "-graphqlfile <path>       "
                                 + Constant.messages.getString("graphql.cmdline.file.help"));
         arguments[ARG_IMPORT_URL_IDX] =
                 new CommandLineArgument(
                         "-graphqlurl",
                         1,
                         null,
                         "",
-                        "-graphqlurl <url>        "
+                        "-graphqlurl <url>         "
                                 + Constant.messages.getString("graphql.cmdline.url.help"));
         arguments[ARG_END_URL_IDX] =
                 new CommandLineArgument(
                         "-graphqlendurl",
                         1,
                         null,
                         "",
-                        "-graphqlendurl <url>  "
+                        "-graphqlendurl <url>      "
                                 + Constant.messages.getString("graphql.cmdline.endurl.help"));
         return arguments;
     }
 
     @Override
     public void execute(CommandLineArgument[] args) {
-        if (arguments[ARG_IMPORT_FILE_IDX].isEnabled()) {
-            System.out.println("Nothing to see here (yet) :)");
-        }
-        if (arguments[ARG_IMPORT_URL_IDX].isEnabled()) {
-            System.out.println("Nothing to see here (yet) :)");
+        if (arguments[ARG_IMPORT_FILE_IDX].isEnabled()
+                || arguments[ARG_IMPORT_URL_IDX].isEnabled()) {
+            if (!arguments[ARG_END_URL_IDX].isEnabled()) {
+                CommandLine.error("Endpoint URL must be specified.");
+            }
+            String endpointUriStr = arguments[ARG_END_URL_IDX].getArguments().firstElement();
+            URI endpointUri = null;
+            try {
+                new URL(endpointUriStr);
+                endpointUri = new URI(endpointUriStr, true);
+            } catch (Exception e) {
+                CommandLine.error("Invalid Endpoint URL.\n" + e.getMessage());
+            }
+            if (arguments[ARG_IMPORT_FILE_IDX].isEnabled()) {
+                String filePath = args[ARG_IMPORT_FILE_IDX].getArguments().firstElement();
+                File file = new File(filePath);
+                if (!file.exists() || !file.canRead()) {
+                    CommandLine.error("Cannot read file: " + file.getAbsolutePath());
+                }
+                if (!file.isFile()) {
+                    CommandLine.error("Invalid File.");
+                }
+                try {
+                    GraphQlParser.parse(FileUtils.readFileToString(file, "UTF-8"), endpointUri);
```

Comment:
It might worth adding a convenience method, e.g. `GraphQlParser#parse(Path, String)`, to avoid duplication.